src/components/menu/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import {
+  Button,
+  Menu as ChakraMenu,
+  MenuItem as ChakraMenuItem,
+  MenuButton,
+  MenuList,
+} from '@chakra-ui/react';
+
+import { ChevronIcon } from '../icons';
+
+interface MenuItem {
+  text: string;
+  onClick: () => void;
+}
+
+export interface MenuProps {
+  title: string;
+  items: MenuItem[];
+}
+
+const Menu: FC<MenuProps> = ({ title, items }) => {
+  return (
+    <ChakraMenu>
+      <MenuButton
+        as={Button}
+        padding={0}
+        rightIcon={<ChevronIcon boxSize="xs" />}
```

Comment:
I cannot figure it out a 100% with the design you have but I think the common pattern is to rotate this chevron when the menu is open?

Reply:
see: https://github.com/smg-automotive/components-pkg/pull/277#discussion_r991916015